src/common/selectors.js

```diff
@@ -408,3 +408,23 @@ export const createGetObjectMessages = objectSelector =>
 // const object = getObject(store.getState(), objectId)
 // ...
 export const getObject = createGetObject((_, id) => id)
+
+export const createGetHostMetrics = hostSelector => _createCollectionWrapper(
+  create(
+    hostSelector,
+    hosts => {
+      const metrics = {
+        count: size(hosts),
```

Comment:
If `hosts` is a map, which is usually the case, `lodash/size` will iterate over it to count the number of properties, while you can simply increments the count in your `forEach`.